### PR TITLE
feat(refactor): behaviour save order and sync account --run-build

### PR DESCRIPTION
**Pull request by jokomyn**
changelog:
- refactor behaviour websocket
- sync account progress modal
- adjustment book table and order page karena update api

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=16) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=16)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=16)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=16)

- Code Smells: security_a [1 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=16)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=16&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage:  [0.00% Coverage](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_coverage&view=list&pullRequest=16)

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=16)

File: src/setup/websocket/SetupWebsocket.ts [hunk 1]
```diff
@@ -1 +1,23 @@
-export const ws = new WebSocket('ws://0.0.0.0:9002')
\ No newline at end of file
+import {checkJson} from 'src/app/utils/cek-json-utils'
```
Comment (akarintitech) [inline src/setup/websocket/SetupWebsocket.ts:1]: **Remove this unused import of 'checkJson'.**  [<sub>Why is this an issue?</sub>](https://sonar.akarinti.tech/coding_rules?open=typescript:S1128&rule_key=typescript:S1128) 

CODE_SMELL Codesmell　　MINOR **Minor**　　:hourglass: *2min* effort

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=16) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=16)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=16)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=16)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=16)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=16&inNewCodePeriod=true)

- Closed Issues: 1 issues closed

## Coverage and Duplications
- Coverage:  [0.00% Coverage](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_coverage&view=list&pullRequest=16)

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=16)

**Review by aldiz1711 — APPROVED** [review]
OK

**Pull request merged**